data/core/units.cfg

```diff
@@ -247,7 +247,24 @@ The Dunefolk’s inquisitive and explorative nature does not preclude military s
         male_name= _ "race^Saurian"
         female_name= _ "race+female^Saurian"
         plural_name= _ "race^Saurians"
-        description= "" # wmllint: ignore
+        description= "Saurians are small, slender, and lizard-like creatures. Much smaller than humans, they rarely stand taller than a ten year old child, though the distance from the tip of their snout to the end their tail can be as long as the average man is tall. Light and nimble, the warriors prefer to fight as they hunt- slipping through enemy lines to target the weak and the injured while their slender figures allow them to evade their attackers.
```

Comment:
The hyphen in "as they hunt-" should be an em dash (&mdash;) or a colon.

Reply:
"as they hunt: slipping" (colon)

or 

"as they hunt—slipping" (em dash)

or

"as they hunt – slipping" (en dash)

I'm not sure if Wesnoth has a guide about whether to prefer the em or en dash.

Reply:
Also, I think I'd prefer "small, slender, lizard-like creatures", without the explicit conjunction. Though the "small" is also redundant with the next sentence... and "slender" is also repeated later on... maybe the first sentence should be rephrased or even mostly-removed?

"to the end of their tail" - missing a preposition

Reply:
> I'm not sure if Wesnoth has a guide about whether to prefer the em or en dash.

data/campaigns/ has plenty of uses of em dash and none of en dash.